tests/QRamTest.qs

```diff
@@ -114,6 +114,151 @@
         Message("Test passed.");
     }
 
+    @Test("QuantumSimulator")
+    operation BBQRAMforTwoMemoryRegistersUnshuffled() : Unit {
+        let m0 = (0, true);
+        let m1 = (1, false);
+        let queryAddress = 0;
```

Comment:
This should be an argument to the test helper function, or something the test loops over and tests each of addresses for a single test.